### PR TITLE
[9.0][IMP] purchase_request_to_rfq: Add supplier to wizard default values all lines have the same supplier

### PR DESCRIPTION
**Pull request by sergio-teruel**
This PR is a minor improvement, when the user selects lines with the same supplier and create a rfq set this supplier as default value in the wizard.

cc @Tecnativa
@jbeficent What do you think of this improvement?
Thanks.

### CODE REVIEW

File: purchase_request_to_rfq/wizard/purchase_request_line_make_purchase_order.py [hunk 1]
```diff
@@ -100,8 +104,13 @@ def default_get(self, fields):
 
         items = []
         self._check_valid_request_line(request_line_ids)
+        supplier_ids = []
         for line in request_line_obj.browse(request_line_ids):
             items.append([0, 0, self._prepare_item(line)])
+            supplier_ids.append(line.supplier_id.id) \
```
Comment (pedrobaeza) [inline purchase_request_to_rfq/wizard/purchase_request_line_make_purchase_order.py]: Use 
```
lines = request_line_obj.browse(request_line_ids)
for line in lines:
    ...
suppliers = lines.mapped('supplier_id')
if len(suppliers) == 1:
    ...
```

**Comment by sergio-teruel** [comment]
@pedrobaeza Changes done, I did not want to modify the original method.

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@sergio-teruel, you should modify certain code lines when there's a great improvement in readibility or efficiency.

**Comment by JordiBForgeFlow** [comment]
:+1: Thanks Sergio! Good contribution!

**Pull request merged**